### PR TITLE
Documents stats mode in --help + changes the --help structure

### PR DESCRIPTION
**Pull request by brandelune**
## Pull request type

- [ ] Bug fix
- [ ] Feature
- [x] Documentation
- [ ] Build and release changes
- [ ] Other (describe below)

## Which ticket is resolved?

https://sourceforge.net/p/omegat/documentation/377/

## What does this PR change?

adds documentation strings to --help + adapt structure

### CODE REVIEW

File: src/org/omegat/Bundle.properties [hunk 1]
```diff
@@ -82,75 +82,100 @@ app-version-template-pretty={0} {1}
...
+#    \n\
+#    Commands:\n\
+#        \t-h, --help\n\
+#            \t\tShow usage information.\n\
+#        \tinit <source language> <target language>\n\
+#            \t\tInitialize a minimal team project with the specified languages.\n\
+#            \t\tIf invoked in a git working tree or svn checkout, the results\n\
+#            \t\twill be staged/added.\n\
+#    \n\
+#    {0}
```
Comment (miurahr) [inline src/org/omegat/Bundle.properties:178]:
```suggestion
TEAM_TOOL_HELP=Usage: java -jar OmegaT.jar team <command> [<options>]]\n\
    \n\
    Commands:\n\
        \t-h, --help\n\
            \t\tShow usage information.\n\
        \tinit <source language> <target language>\n\
            \t\tInitialize a minimal team project with the specified languages.\n\
            \t\tIf invoked in a git working tree or svn checkout, the results\n\
            \t\twill be staged/added.\n\
    \n\
    {0}
```

Comment (brandelune) [reply src/org/omegat/Bundle.properties:178]: I don't think it is necessary since I put the explanations between line 104 and 108.

Comment (miurahr) [reply src/org/omegat/Bundle.properties:178]: CI test failed with `java.util.MissingResourceException: Can't find resource for bundle java.util.PropertyResourceBundle, key TEAM_TOOL_HELP`

Comment (brandelune) [reply src/org/omegat/Bundle.properties:178]: Ok. Sorry. I'll looking for the place in the code where this is expected. Unless you have a strong opinion about that help key, I'd rather remove it from the code.

**Comment by miurahr** [comment]
Squash and merged into `master`

**Pull request closed without merging**